code/game/gamemodes/vampire/vampire.dm

```diff
@@ -297,22 +297,23 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 			to_chat(owner, "<span class='warning'>They've got no blood left to give.</span>")
 			break
 		if(H.stat < DEAD)
-			if(!issmall(H) && !H.ckey)
+			if(!issmall(H) || H.ckey)
 				blood = min(20, H.blood_volume)	// if they have less than 20 blood, give them the remnant else they get 20 blood
 				bloodtotal += blood / 2	//divide by 2 to counted the double suction since removing cloneloss -Melandor0
 				bloodusable += blood / 2
 		else
-			if(!issmall(H) && !H.ckey)
+			if(!issmall(H) || H.ckey)
 				blood = min(5, H.blood_volume)	// The dead only give 5 blood
 				bloodtotal += blood
 		if(old_bloodtotal != bloodtotal)
-			if(!issmall(H) && !H.ckey)
+			if(!issmall(H) || H.ckey) // not small OR has a ckey, monkeys with ckeys can be sucked, humanized monkeys can be sucked monkeys without ckeys cannot be sucked
 				to_chat(owner, "<span class='notice'><b>You have accumulated [bloodtotal] [bloodtotal > 1 ? "units" : "unit"] of blood[bloodusable != old_bloodusable ? ", and have [bloodusable] left to use" : ""].</b></span>")
 		check_vampire_upgrade()
 		H.blood_volume = max(H.blood_volume - 25, 0)
 		if(ishuman(owner))
 			var/mob/living/carbon/human/V = owner
 			if(issmall(H) && !H.ckey)
+				to_chat(V, "<span class='notice'><b>feeding on [H] reduces your hunger, but you get no usable blood from it.</b></span>")
```

Comment:
Sorry to keep nitpicking, this one's my fault for bothering you to change it to begin with haha.

The 'F' at the beginning should be capitalized here